apis/slo/v1alpha1/pod.go

```diff
@@ -64,3 +65,49 @@ func GetPodMemoryQoSConfig(pod *corev1.Pod) (*PodMemoryQOSConfig, error) {
 	}
 	return &cfg, nil
 }
+
+const (
+	// AnnotationPodEnableCoreSched is the annotation key of the Linux Core Scheduling enabled.
+	// Value "true" means the core sched is explicitly enabled.
+	// Value "false" means the core sched is explicitly disabled.
+	// Other values or missing the annotation means the status follows the node-level settings.
+	AnnotationPodEnableCoreSched = apiext.DomainPrefix + "enable-core-sched"
+
+	// AnnotationCoreSchedGroupID is the annotation key of the group ID of the Core Sched.
+	// Value is expected to be a valid UUID.
+	// Core Sched: https://docs.kernel.org/admin-guide/hw-vuln/core-scheduling.html
+	// NOTE: Pods with the different core sched group IDs will not be running at the same SMT core at the same time,
+	//       which means they will take different core sched cookies. If a pod sets the core sched disabled, it will
+	//       take the default core sched cookie (0) and will also be force-idled to run on the same SMT core
+	//       concurrently with the core-sched-enabled pods.
+	//       If a pod sets the core sched enabled but misses the core sched ID, the pod will be assigned an individual
+	//       and stateless internal ID according to the pod's UID.
+	AnnotationCoreSchedGroupID = apiext.NodeDomainPrefix + "/core-sched-group-id"
```

Comment:
The annotation's prefix is NodeDomainPrefix, but only Pods can use the key, so is this a typo? Also I would recommend merging these two annotations.